.travis.yml

```diff
@@ -1,8 +1,8 @@
 sudo: false
 language: cpp
-dist: xenial
 
-if: type != push OR branch =~ /^v\d+\.\d+(\.\d+)?(-\S*)?$/
```

Comment:
You removed the code above. It this for #504 ?

Reply:
Yes